### ISSUE TITLE
don't squelch npm logging

### ISSUE THREAD

**Issue by boneskull** [spion/npm-runtools#1]
Hi,

This module was exactly what I was searching for.  However, it was difficult to verify the tasks had actually been executed, because of the presumption of `--loglevel silent`.  If users wish to quiet the logs, they should be able to do this on a per-task basis.

If you agree, I'll send a PR.

**Comment by boneskull** [comment]
sending it anyway

**Comment by spion** [comment]
Sorry, I forgot to reply to this PR earlier. Logs were silenced on purpose, but they are only npm logs, not the application logs.

What is your intended use case? Is it to figure out which command(s) have failed? If so, we should add custom logging for that on exit. I'd say npm's default log level is unnecessarily verbose and it only gets worse when running tasks in parallel.

**Comment by boneskull** [comment]
> What is your intended use case?

I want to ensure the tasks are run by inspecting their output.

> I'd say npm's default log level is unnecessarily verbose and it only gets worse when running tasks in parallel.

That's a valid opinion.  But others (like myself) don't share it.

AFAIK you can force current behavior with `parallel --loglevel=silent` anyway.

**Comment by boneskull** [comment]
If this is to be a drop-in replacement for `npm run-script`, then `parallel` should probably _not_ override `npm`'s defaults.

**Comment by spion** [comment]
@boneskull but thats not suppressing the task output, only npm's output. Task output is still properly redirected to stdout/stderr. Is there any particular part of npm's output that you need? Last time I checked it only showed the commands being run.

**Comment by boneskull** [comment]
Ah.  I recall now.  Yes, I want to see the commands being run.  Still, I think this is important:

> If this is to be a drop-in replacement for npm run-script, then parallel should probably not override npm's defaults.

**Comment by boneskull** [comment]
iirc you can use `~/.npmrc` to make loglevel=silent the default

**Comment by spion** [comment]
But I don't want to make the loglevel silent - I do want the logging, I just don't want it for every single task being run as part of the parallel block. Once I use `parallel` on the tasks I want for npm to treat them as one task.

At the very least I'd like some predictability. IIRC the order of npm's messages (multiple lines are being logged) is not consistent - at least not on my system.

I'll give this a try and will let you know what bothers me about it - tbh I can't recall the specific reason why I removed the logging.

**Comment by boneskull** [comment]
Yes, the ordering won't consistent.  I understand that it can be annoying, but if I have `parallel` running `parallel` running `parallel` (and, actually, I think I do), then I want to make sure everything's happening.
